spec/features/review/user_deletes_review_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+feature 'user deletes their review', %{
+  As an authenticated user
+  I want to be able to delete my review
+  Because I no longer want that review on the site
+} do
+
+  # [] I must be signed in
+  # [] If I'm not signed in, I should be prompted to do so
+  # [] I can only delete my own reviews
+  # [] I delete by clicking on the delete button in the Edit Profile page
+  # [] I am prompted to confirm the deletion
+  # [] I get a successfully deleted message
+  # [] I no longer see my review on the Restaurant's show page
+
+  context "user is signed in" do
+    let!(:restaurant) { FactoryGirl.create(:restaurant) }
+    let!(:user) { FactoryGirl.create(:user) }
+    
+    scenario 'an authenticated user wants to delete their review' do
+      sign_in_as(user)
+      let!(:review) { FactoryGirl.create(:review, restaurant_id: restaurant.id,
+        user_id: user.id) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 24, 27 should be on its own line.